scripts/prepare_heat_demand.py

```diff
@@ -0,0 +1,305 @@
+    sectors
+
+    Parameters
+    ----------
+    region : str
+        region e.g. BB as Brandenburg
+    scenario : str
+        scenario e.g. "base"
+
+    Returns
+    -------
+    heat_load_total : pd.DataFrame
+         DataFrame that contains the calculated total heat load
+
+    """
+    # Set examined years
+    years = np.arange(2010, 2020)
+
+    # Read time series template
+    ts_template = dp.load_b3_timeseries(in_path5, sep=";")
+
+    # Add empty data frame for results / output
+    heat_load = pd.DataFrame(columns=ts_template.columns)
+
+    for index, year in enumerate(years):
+        # Get holidays
+        holidays = get_holidays(in_path3, year, region)
+
+        # Read weather data
+        path_weather_data = os.path.join(
+            in_path1, "weatherdata_" + region + "_" + str(year) + ".csv"
+        )
+        temperature = pd.read_csv(path_weather_data, usecols=["temp_air"], header=0)
+
+        # Get heat demand in region and scenario
+        yearly_demands, sc_demand_unit = get_heat_demand(in_path4, region, scenario)
+
+        types = ["central", "decentral"]
+
+        # Add DataFrame time index for demands
+        demand = pd.DataFrame(
+            index=pd.date_range(
+                datetime.datetime(year, 1, 1, 0), periods=len(temperature), freq="H"
+            )
+        )
+
+        for type in types:
+            # Calculate sfh (efh: Einfamilienhaus) heat load
+            demand["efh" + "_" + type] = bdew.HeatBuilding(
+                demand.index,
+                holidays=holidays,
+                temperature=temperature,
+                shlp_type="EFH",
+                building_class=6,
+                wind_class=0,
+                annual_heat_demand=yearly_demands["efh" + "_" + type],
+                name="EFH",
+                ww_incl=True,
+            ).get_bdew_profile()
+
+            # Calculate mfh (mfh: Mehrfamilienhaus) heat load
+            demand["mfh" + "_" + type] = bdew.HeatBuilding(
+                demand.index,
+                holidays=holidays,
+                temperature=temperature,
+                shlp_type="MFH",
+                building_class=2,
+                wind_class=0,
+                annual_heat_demand=yearly_demands["mfh" + "_" + type],
+                name="MFH",
+                ww_incl=True,
+            ).get_bdew_profile()
+
+            # Calculate industry, trade, service (ghd: Gewerbe, Handel, Dienstleistung)
+            # heat load
+            demand["ghd" + "_" + type] = bdew.HeatBuilding(
+                demand.index,
+                holidays=holidays,
+                temperature=temperature,
+                shlp_type="ghd",
+                wind_class=0,
+                annual_heat_demand=yearly_demands["ghd" + "_" + type],
+                name="ghd",
+                ww_incl=True,
+            ).get_bdew_profile()
+
+        # Calculate total heat load in year
+        heat_load_year = pd.DataFrame(
```

Comment:
The `var_name` should be `heat_central-load-profile` or `heat_decentral-load-profile`.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/ae095e268fdc66e2277c389abf1a33e7abe29270.